vsts-fullbuild-ubuntu-vm/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Ubuntu VM with cross-platform SDKs and Team Services agent",
```

Comment:
OK, updated.  I did the best I could in the 60 char. limit and still mention Ubuntu and Team Services for search engine optimization.  Thanks!